roles/collection/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+# Update or destroy Automation Hub Collection
+- name: Update or destroy Automation Hub Collection
+  collection:
+    namespace:             "{{ __collection.namespace }}"
+    name:         "{{ __collection.name }}"
+    version:      "{{ __collection.version | default('') }}"
+    path:          "{{ __collection.path | default('') }}"
+    wait:            "{{ __collection.wait | default('true') }}"
+    auto_approve:       "{{ __collection.auto_approve | default('true') }}"
+    overwrite_existing:        "{{ __collection.overwrite_existing | default('true') }}"
+    state:            "{{ __collection.state | default('present') }}"
+    ah_host:          "{{ ah_host | default(ah_hostname) }}"
+    ah_username:      "{{ ah_username | default(omit) }}"
+    ah_password:      "{{ ah_password | default(omit) }}"
+    ah_token:         "{{ ah_token | default(omit) }}" ###??????? Do all of the roles want this now
```

Comment:
Only some roles can take this. It depends on what the module can take.

The module has the line

```
extends_documentation_fragment: redhat_cop.ah_configuration.auth
```

which points to `doc_fragments/auth.py` which does have the `ah_token` endpoint

Reply:
Can you remove this comment `###??????? Do all of the roles want this now`